docs/two-phase-commit-transactions.md

```diff
@@ -301,6 +304,15 @@ tx.getId();
 
 <div class="notice--info">{{ notice--info | markdownify }}</div>
 
+In case of using `join()` to re-join a transaction, if you have not joined the transaction before, it returns a new transaction object. On the other hand, in case of using `resume()` to resume a transaction, if you have not joined the transaction before, it throws `TransactionNotFoundException`.
+
+{% capture notice--info %}
+**Note**
+
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
```

Comment:
This note section is empty. Is it OK to delete it?

```suggestion
```

Reply:
Ah good catch! I wanted to add the sentence `When using join() to re-join ...` as a note. I've fixed it in fcc372dd8f4458cb89dff05aa99026f35b59ae69. Thanks.